daemon/daemon.go

```diff
@@ -760,9 +760,23 @@ func (daemon *Daemon) Shutdown() error {
 
 				go func() {
 					defer group.Done()
-					// If container failed to exit in 10 seconds of SIGTERM, then using the force
-					if err := c.Stop(10); err != nil {
-						logrus.Errorf("Stop container %s with error: %v", c.ID, err)
+					if c.IsPaused() {
+						logrus.Debugf("container %s is paused, send 15 to container before unpause it", c.ID)
+						if err := daemon.Kill(c, 15); err != nil {
```

Comment:
Please use preferrably `pkg/signal`'s map to get SIGTERM. You might need to do some adjustments in that package wrt to windows.